src/playbacks/html5_video/html5_video.js

```diff
@@ -84,16 +84,19 @@ export default class HTML5Video extends Playback {
     this._stopped = false
     this._options = options
     this._setupSrc(options.src)
-    this.el.loop = options.loop
-    if (options.poster) {
-      this.$el.attr("poster", options.poster)
-    }
-    this.el.autoplay = options.autoPlay
-    if (Browser.isSafari) {
-      this._setupSafari()
-    } else {
-      this.el.preload = options.preload ? options.preload: 'metadata'
-    }
+
+    var playbackConfig = (options.playbackConfig || {})
+    var preload = playbackConfig.preload || ((Browser.isSafari)?'auto':options.preload)
+
+    setupObject(this.el, {
+      loop: playbackConfig.loop || options.loop,
+      autoplay: playbackConfig.autoplay || options.autoPlay,
+      poster: playbackConfig.poster || options.poster,
+      preload: preload || 'metadata',
+      controls: playbackConfig.preload || options.useVideoTagDefaultControls,
+      crossorigin: playbackConfig.crossorigin
+    })
```

Comment:
This works different to be before in that it is setting the properties on the video tag instead of adding attributes to the dom element which I guess is fine, except for poster which looks like it can only be set as an attribute.